rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.75.0"
```

Comment:
This can cause troubles in our release images, Soph attempted it before: https://github.com/prisma/prisma-engines/pull/4611#issuecomment-1875496172